app/src/main/java/net/squanchy/speaker/view/SpeakerAdapter.java

```diff
@@ -0,0 +1,44 @@
+package net.squanchy.speaker.view;
+
+import android.support.annotation.Nullable;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.Collections;
+import java.util.List;
+
+import net.squanchy.R;
+import net.squanchy.speaker.Speaker;
+
+public class SpeakerAdapter extends RecyclerView.Adapter<SpeakerViewHolder> {
+
+    private List<Speaker> speakerList = Collections.emptyList();
+    private SpeakersView.OnSpeakerClickedListener listener;
+
+    public SpeakerAdapter() {
+    }
+
+    public void updateWith(List<Speaker> speakerList, @Nullable SpeakersView.OnSpeakerClickedListener listener) {
+        this.speakerList = speakerList;
+        this.listener = listener;
+        notifyDataSetChanged();
+    }
+
+    @Override
+    public SpeakerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_speaker, parent, false);
```

Comment:
Don't get the context from the parent! Use this view's or you risk picking up the wrong theme if there's one specified ⚠️